### PR TITLE
Update Smartgit to 6.5.5

### PR DESCRIPTION
**Pull request by tmoreira2020**

### CODE REVIEW

File: Casks/smartgit.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask :v1 => 'smartgit' do
-  version '6.5.2'
-  sha256 '3fc0fd89e1e2b7e5aec5f12b8da2ae1da1d2a84d96ce335a62d922db1640b965'
+  version '6.5.5'
+  sha256 '5736c91b8b05f19c54b4b89aab7f83db1516da2098ba041bd37ffe4403481872'
 
   url "http://www.syntevo.com/download/smartgit/smartgit-macosx-#{version.gsub('.', '_')}.dmg"
```
Comment (Amorymeltzer) [inline Casks/smartgit.rb]: Apparently the url has changed slightly, using "download**s**" instead of the singular, would you mind changing it here?

**Comment by tmoreira2020** [comment]
Done

**Comment by Amorymeltzer** [comment]
Great, thanks.  Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by tmoreira2020** [comment]
I will close this PR and open two others to handle the issues separate.

**Pull request closed without merging**